### PR TITLE
Bug 1140583 - use <bdi> to wrap numbers

### PR DESCRIPTION
**Pull request by EragonJ**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[EragonJ EragonJ (E.J.) (EragonJ)](https://github.com/EragonJ) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=6cd0ca5f60e9)

**Comment by try-server-hook** [comment]
[EragonJ EragonJ (E.J.) (EragonJ)](https://github.com/EragonJ) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=1aaf52521801)

**Comment by try-server-hook** [comment]
[EragonJ EragonJ (E.J.) (EragonJ)](https://github.com/EragonJ) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=347c8a36c0d3)

File: apps/settings/js/panels/about_more_info/device_info.js [hunk 1]
```diff
@@ -88,8 +88,16 @@ define(function(require) {
...
-          span.textContent = (imeis.length > 1) ?
-            'IMEI ' + (index + 1) + ': ' + imei : imei;
+          if (imeis.length > 1) {
+            navigator.mozL10n.setAttributes(span,
+              'deviceInfo-IMEI-with-index', {
+                index: index + 1,
+                imei: imei
+            });
+          } else {
+            span.textContent = imei;
```
Comment (crh0716) [inline apps/settings/js/panels/about_more_info/device_info.js]: We should remove l10n attribute at the same time.

File: apps/settings/js/panels/about_more_info/device_info.js [hunk 2]
```diff
@@ -150,12 +158,21 @@ define(function(require) {
...
-          span.textContent = multiSim ?
-            'SIM ' + (index + 1) + ': ' + conn.iccId : conn.iccId;
+          if (multiSim) {
+            navigator.mozL10n.setAttributes(span,
+              'deviceInfo-ICCID-with-index', {
+                index: index + 1,
+                iccid: conn.iccId
+            });
+          } else {
+            span.textContent = conn.iccId;
```
Comment (crh0716) [inline apps/settings/js/panels/about_more_info/device_info.js]: same here.

File: apps/settings/js/panels/about_more_info/device_info.js (hunk 1 above)
Comment (EragonJ) [inline apps/settings/js/panels/about_more_info/device_info.js]: based on our offline discussion, we can ignore this line

File: apps/settings/js/panels/about_more_info/device_info.js (hunk 2 above)
Comment (EragonJ) [inline apps/settings/js/panels/about_more_info/device_info.js]: based on our offline discussion, we can ignore this line

**Comment by try-server-hook** [comment]
[EragonJ EragonJ (E.J.) (EragonJ)](https://github.com/EragonJ) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=2bcfcabc746f)

**Pull request merged**